### PR TITLE
Rewrite top-level README in preparation for documenting each component.

### PR DESCRIPTION
**Pull request by lgolding**
An almost complete rewrite.

### CODE REVIEW

**Comment by lgolding** [comment]
@michaelcfanning is added to the review. #Closed

**Comment by lgolding** [comment]
@EasyRhinoMSFT is added to the review. #Closed

**Comment by lgolding** [comment]
@rtaket is added to the review. #Closed

**Comment by lgolding** [comment]
@cfaucon is added to the review. #Closed

File: README.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
-* Write a `JsonSchema` object to a stream.
-* Generate .NET classes from a schema.
-* Validate the generated classes against the schema.
+The JSchema repo consists of the following components, each of which is documented in its own README.md:
+
+* `Microsoft.Json.Pointer`: an implementation of the JSON Pointer specification ([RFC 6901](https://tools.ietf.org/html/rfc6901)). Documentation: coming soon.
+
+* `Microsoft.Json.Schema`: an almost but not quite complete implementation of [JSON Schema Draft 4](http://json-schema.org/specification-links.html#draft-4). Documentation: coming soon.
+
+* `Microsoft.Json.Schema.Validation`: a library to validate a JSON instance document against a JSON schema document.  Documentation: coming soon.
```
Comment (EasyRhin0) [inline README.md]: >  [](start = 117, length = 1)

Extra space

Comment (lgolding) [reply README.md]: Thanks, fixed.

---
In reply to: [218613258](https://github.com/Microsoft/jschema/pull/47#discussion_r218613258) [](ancestors = 218613258)

**Pull request merged**